GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -504,19 +505,19 @@ public GVRSceneObject getAssimpModel(String assetRelativeFilename,
         return GVRImporter.getAssimpModel(this, assetRelativeFilename, settings);
     }
 
-    public GVRSceneObject loadJassimpModelFromSD(String externalFile) throws IOException {
+    public GVRModelSceneObject loadJassimpModelFromSD(String externalFile) throws IOException {
```

Comment:
changing the return type to GVRModelSceneObject will break API compatibility.  maybe this is the time to introduce that loadModel() api discussed in another check-in.  That way we could introduce that new api and deprecate the loadJassimp* ones.  We will be able to remove the loadJassimp* apis in a future revision.